pre-lollipop-activity-transition/src/main/java/com/kogitune/activity_transition/core/TransitionAnimation.java

```diff
@@ -18,7 +18,10 @@
  */
 public class TransitionAnimation {
     private static final String TAG = "Transition";
+    private static final int MAX_TIME_TO_WAIT = 3000;
+    public static final Object lock = new Object();
     public static WeakReference<Bitmap> bitmapCache;
+    public static boolean isReady = false;
```

Comment:
I think `isReady` variable name is abstract. 
I suggest to use `isImageFileReady`.